src/main/js/main_html5.js

```diff
@@ -1514,7 +1514,13 @@ require("../../../html5-common/js/utils/environment.js");
      * @method OoyalaVideoWrapper#raiseRatechangeEvent
      */
     var raiseRatechangeEvent = _.bind(function() {
-      this.controller.notify(this.controller.EVENTS.RATE_CHANGE);
+      if (!_video) {
```

Comment:
I was thinking of Bitrate change not playbackrate change. You can disregard this. Although maybe we should rename this to playback_rate_change?

Reply:
I don't think this will actually happen, it's mostly just to prevent an uncaught exception in case of an edge case. I think the event might not be that useful without the `playbackRate`, but another option would be to fire it with a null value. I think the parameter should always be there for consistency

Reply:
@pilievOoyala yes, I think playback_rate_change would be better. Will rename